src/popup.ts

```diff
@@ -86,53 +86,103 @@ function getContent (notif: Notification): string {
 	}
 }
 
-function getAuthorNote (notif: Notification): string {
+function getAuthorNote (notif: Notification): Node {
+	const noteTag = document.createElement("p");
+	noteTag.className = "author-note";
+	const authorTag = document.createElement("b");
+	const titleTag = document.createElement("b");
+
 	if (notif.modNickname) {
 		/* Moderator Message */
-		return `<b>${escapeHTML(notif.modNickname)}</b> sent you a guardian message:`;
+		authorTag.innerHTML = notif.modNickname;
```

Comment:
You need to use `escapeHTML()` when setting innerHTML.

Reply:
Like Matthias pointed out, just use textContent instead of innerHTML, and no need for escaping.